specification/caas-published-cohort-definitions.yaml

```diff
@@ -27,6 +27,188 @@ info:
+
+    ### You can try out the sandbox importing our Postman collection:
+    [published-cohort-definitions.postman_collection.json](../tests/postman-collection/published-cohort-definitions.postman_collection.json)
+    
+    ```json
+    {
+      "info": {
+        "_postman_id": "5ce2b578-3079-4ea9-a33e-06ed64a0ccb2",
+        "name": "published-cohort-definitions",
+        "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+        "_exporter_id": "18640524"
+      },
+      "item": [
+        {
+          "name": "PublishedCohortLibraryGetBySlugName - SANDBOX",
+          "request": {
+            "method": "POST",
+            "header": [],
+            "body": {
+              "mode": "graphql",
+              "graphql": {
+                "query": "query PublishedCohortLibraryGetBySlugName {\n    PublishedCohortLibraryGetBySlugName(urlSlug: \"flu-22-to-23\") {\n        ... on Cohort {\n            id\n            name\n            urlSlug\n            conditionGroups {\n                id\n                name\n                urlSlug\n            }\n        }\n    }\n}\n",
+                "variables": ""
+              }
+            },
+            "url": {
+              "raw": "https://internal-dev-sandbox.api.service.nhs.uk/caas-published-cohort-definitions/api",
+              "protocol": "https",
+              "host": [
+                "internal-dev-sandbox",
+                "api",
+                "service",
+                "nhs",
+                "uk"
+              ],
+              "path": [
+                "caas-published-cohort-definitions",
+                "api"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "PublishedCohortLibraryGetBySlugName",
+          "request": {
+            "auth": {
+              "type": "apikey",
+              "apikey": [
+                {
+                  "key": "value",
+                  "value": "lVlQRHlM4M111q8fnmLBe201HAWMNQJH16SU8Q4C",
+                  "type": "string"
+                },
+                {
+                  "key": "key",
+                  "value": "x-api-key",
+                  "type": "string"
+                }
+              ]
+            },
+            "method": "POST",
+            "header": [],
+            "body": {
+              "mode": "graphql",
+              "graphql": {
+                "query": "query PublishedCohortLibraryGetAll {\n    PublishedCohortLibraryGetAll {\n        authors\n        clinicalAtRiskGroupsText\n        id\n        commissioner\n        demographicsText\n        description\n        disclaimerText\n        fixedDateReference\n        name\n        purpose\n        shortName\n        summary\n        urlSlug\n    }\n}",
+                "variables": ""
+              }
+            },
+            "url": {
+              "raw": "https://internal-dev.api.service.nhs.uk/caas-published-cohort-definitions/api",
```

Comment:
Why there are both sandbox and apim postman tests?

I think we should stick only with the sandobx ones

Reply:
```
As part of developing an API for API-M it's recommended that we produce a Postman collection for our API. This is a set of preconfigured queries that 3rd parties can use to try out our API.

A collection of Postman queries, importable by anyone, that connect to the API and can be run without needing to be modified

Use Newman to test the Postman collection
```

The tickets talks about our API so I assumed the published cohort definitions and sandbox but I'm happy to have the sandbox only if it's better

Reply:
✅ removing apim postman test